### PR TITLE
sawblade prefab added

### PR DESCRIPTION
**Pull request by Gowtham100**

### CODE REVIEW

File: Assets/Scripts/MovingPlatform.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    public int pointSelection;
+
+	// Use this for initialization
+	void Start () {
+        currentPoint = points[pointSelection];
+	}
+	
+	// Update is called once per frame
+	void Update () {
+        platform.transform.position = Vector3.MoveTowards(platform.transform.position, currentPoint.position, Time.deltaTime*moveSpeed);
```
Comment (jayfriso) [inline Assets/Scripts/MovingPlatform.cs]: We should use Vector2 and not Vector 3

**Comment by jeaniewu** [comment]
solve the merge conflict first, and also @Gowtham100 give a description on how to use this prefab, like if u drag it to the scene, will it work right away?

**Comment by Gowtham100** [comment]
Im not sure what the conflict is. Cuz the code seems yo be okay. Im guessing its version difference? Mine is .4?  And yes! You just drag and drop it.

**Comment by jeaniewu** [comment]
Pull from master and resolve it in your local, and then push another patch

File: Assets/Scripts/MovingPlatform.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        if(platform.transform.position == currentPoint.position)
+        {
+            pointSelection++;
+
+            if (pointSelection == points.Length)
+            {
+                pointSelection = 0;
+            }
+
+            currentPoint = points[pointSelection];
```
Comment (jeaniewu) [inline Assets/Scripts/MovingPlatform.cs:30]: what are these points for?

**Comment by Gowtham100** [comment]
Thatsfor the sawblade if it goes to multiple points in the future if we need em. So far we just need two poijts but i thought its better to put it in an array. And yeah i did pull from master...it works for me

**Comment by Gowtham100** [comment]
Lemme try doing that again hopefully no conflict this time

**Comment by Gowtham100** [comment]
hmm yeah Im having problems with the versions. Is yours 5.3.3 or 5.3.4? Cuz mine is 5.3.4

**Comment by jeaniewu** [comment]
you can just not push the file with version, or resolve the conflict by putting your version in

**Comment by jeaniewu** [comment]
I still don't get why you need points

**Pull request closed without merging**